data/SAbDab_Chen/meta.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: SAbDab_Chen
+description: "Antibody data from Chen et al, where they process from the SAbDab. \n        From an initial dataset of 3816 antibodies, they retained 2426\
+    \ antibodies\n        that satisfy the following criteria: 1. \n        have both sequence (FASTA) and Protein Data Bank (PDB) structure files,\n  \
+    \      2. contain both a heavy chain and a light chain, and 3. \n        have crystal structures with resolution < 3 Å. \n        The DI label is derived\
+    \ from BIOVIA's pipelines."
```

Comment:
the linebreaks seems a bit awkward, do you have an idea where they come from?

Reply:
I guess that was the Ångström Å!

Reply:
I converted to nm.